po/pl.po

```diff
@@ -139,10 +141,14 @@ msgid ""
 "\n"
 "Note that forcing an application to quite might cause data loss."
```

Comment:
This needs fixing: _quite_ -> _quit_ :-)

Reply:
I’ll update the PR when #270 gets merged.